graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationFormContainer.tsx

```diff
@@ -107,6 +108,7 @@ class EventNotificationFormContainer extends React.Component<
   handleSubmit = () => {
     const { action, embedded, onSubmit, history } = this.props;
     const { notification } = this.state;
+    const currentUser = CurrentUserStore.getInitialState();
```

Comment:
There is a hook `useCurrentUser` that is the recommended way to access the state.